### PR TITLE
feat(Toast): introduce Toast component

### PR DESCRIPTION
**Pull request by mainframev**
[JIRA ticket](https://jira.kiwi.com/browse/ORBIT-1567)

 Storybook: https://orbit-silvenon-feat-toast-component.surge.sh

### CODE REVIEW

File: packages/orbit-components/src/Toast/helpers.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  100% {
+    transform: translateX(0%);
+  }
+}`;
+
+export const resolveFadeAnimations: resolveAnimation = ({ animate, isHovered }) => {
+  if (animate) {
+    return css`
+      animation: ${fadeIn} 1s linear, ${fadeOut} 1s linear 5s forwards;
+      animation-play-state: ${isHovered ? "paused" : "running"};
```
Comment (silvenon) [inline packages/orbit-components/src/Toast/helpers.js:62]: When hovered I think it's best to bring it back completely. Currently it pauses in whatever state it is, even when it's partially faded out.

File: packages/orbit-components/src/Toast/index.jsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+// $FlowFixMe: https://github.com/flow-typed/flow-typed/issues/3653#issuecomment-568539198
+StyledWrapper.defaultProps = {
+  theme: defaultTheme,
+};
+
+const StyledInnerWrapper = styled.div`
+  ${({ theme, animate }) => css`
+    position: relative;
+    cursor: pointer;
```
Comment (silvenon) [inline packages/orbit-components/src/Toast/index.jsx:37]: Why `cursor: pointer`? Nothing happens on click.

**Review by silvenon — CHANGES_REQUESTED** [review]
- I think the "countdown" should happen _after_ Toast faded in, not during.
- I think when hovering the Toast can still be able to expire, but not go away until the mouse is removed, if you understand what I mean.

File: packages/orbit-components/src/Toast/index.jsx (hunk 2 above)
Comment (mainframev) [reply packages/orbit-components/src/Toast/index.jsx:37]: it's because in the first version it's supposed to be clickable. Plus

**Comment by mainframev** [comment]
I decided to change implementation of toast, because currently it looks as a draft. I will reopen it or create a new PR.

**Pull request closed without merging**